lib/IRGen/GenArchetype.cpp

```diff
@@ -361,6 +432,19 @@ const TypeInfo *TypeConverter::convertArchetypeType(ArchetypeType *archetype) {
             ? IsABIAccessible
             : IsNotABIAccessible;
   }
+
+  auto &ASTContext = IGM.getSwiftModule()->getASTContext();
+  if (ASTContext.LangOpts.hasFeature(Feature::BitwiseCopyable)) {
+    // TODO: Should this conformance imply isAddressOnlyTrivial is true?
+    auto *proto = ASTContext.getProtocol(KnownProtocolKind::BitwiseCopyable);
+    // It's possible for the protocol not to exist if the stdlib is built
+    // no_asserts.
+    if (proto && llvm::is_contained(archetype->getConformsTo(), proto)) {
```

Comment:
This will fail if the archetype conforms to a protocol that inherits from BitwiseCopyable. You should instead use ModuleDecl::lookupConformance()

Reply:
Fixed.  It might be nice to add a convenience wrapping that to Type so that some third object didn't have to be consulted.